src/main/java/de/zalando/aruha/nakadi/validation/EventValidation.java

```diff
@@ -59,6 +61,14 @@ private static JSONObject wrapSchemaInData(final JSONObject schema, final EventT
         return wrapper;
     }
 
+    private static void moveDefinitionsToRoot(final JSONObject wrapper, final JSONObject schema) {
```

Comment:
I didn't do so because I wanted to avoid coupling tests to the implementation. I added a test case that shows that "definitions" should work without worrying about what goes on in the internals of validation.

I could improve test case for the effective schema task once it's merged. Wdyt?